src/components/SignerPanel/SignMsgContent.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Info, Text, SidePanelSeparator, theme } from '@aragon/ui'
+
+import SignerButton from './SignerButton'
+import ToggleContent from '../ToggleContent'
+import LocalIdentityBadge from '../IdentityBadge/LocalIdentityBadge'
+import AppInstanceLabel from '../../apps/Permissions/AppInstanceLabel'
```

Comment:
Ok, i did this in its own [commit](https://github.com/aragon/aragon/pull/708/commits/cbca2a16756f0131bdc66c8c7b604127c9efba7c) because it touches a lot of files. Would be really nice to try and get this in before others that touch the same components, so we don't have to keep fixing merge conflicts in this PR

Reply:
Agreed, thanks!